test/samples.js

```diff
@@ -129,18 +129,18 @@ exports.samples = [
         file: 'p2021/2021-rec',
         profile: 'REC',
         title: 'High Resolution Time',
-        docDate: '2021-11-4',
+        docDate: '2021-12-1',
         editorIDs: [56102],
         editorNames: ['George Herald'],
         informative: false,
         process: 'https://www.w3.org/2021/Process-20211102/',
         delivererIDs: [45211],
         charters: ['https://www.w3.org/2021/02/webperf.html'],
         patentPolicy: 'https://www.w3.org/Consortium/Patent-Policy-20200915/',
-        thisVersion: 'https://www.w3.org/TR/2021/REC-hr-time-20211104/',
-        latestVersion: 'https://www.w3.org/TR/hr-time/',
-        history: 'https://www.w3.org/standards/history/hr-time',
-        previousVersion: 'https://www.w3.org/TR/2021/WD-hr-time-3-20211201/',
+        thisVersion: 'https://www.w3.org/TR/2021/WD-hr-time-3-20211201/',
```

Comment:
This test case is called `2021-rec` and it's supposed to run on a Recommendation

Reply:
@jennyliang220 I replaced the test file.